gen_config/model.conf

```diff
@@ -1 +1 @@
-java -cp target/killbill-java-parser-0.0.1-SNAPSHOT.jar com.ning.killbill.KillbillParser -i file:///<KILLBILL_REPO>/jaxrs/src/main/java/com/ -o /tmp/output -t PHP_CLIENT_API -p com.ning.billing.jaxrs.json
+java -cp target/killbill-java-parser-0.0.1-SNAPSHOT.jar com.ning.killbill.KillbillParser -i file://<KILLBILL_REPO>/jaxrs/src/main/java/org -o /tmp/output -t PHP_CLIENT_API -p org.killbill.billing.jaxrs.json
```

Comment:
Indeed, we changed the namespace quite a few years back 😸 

However, looking at the script that I use locally (just verified it still runs correctly), I do have `file:///$BASE` (3 `/`) where `BASE=/foo/bar/baz/killbill/jaxrs/src/main/java/` (no `org`).

Your version may work but that way we would be consistent with others (e.g. [Ruby](https://github.com/killbill/killbill-client-ruby/blob/master/gen_config/model.conf) one).